assets/js/modules/analytics-4/components/dashboard/EnhancedMeasurementActivationBanner/SetupBanner.js

```diff
@@ -0,0 +1,208 @@
+		} );
+
+		const { error } = await submitChanges();
+
+		setIsSaving( false );
+
+		if ( error ) {
+			setErrorNotice( error );
+			return;
+		}
+
+		// Ask the parent component to show the success banner.
+		// This should be called last because it will unmount this component.
+		onSubmitSuccess();
+	}, [ setValues, submitChanges, onSubmitSuccess ] );
+
+	const handleSubmitChanges = async () => {
+		const scopes = [];
+
+		if ( hasEditScope === false ) {
+			scopes.push( EDIT_SCOPE );
+		}
+
+		// If scope not granted, trigger scope error right away. These are
+		// typically handled automatically based on API responses, but
+		// this particular case has some special handling to improve UX.
+		if ( scopes.length > 0 ) {
+			setValues( FORM_SETUP, { autoSubmit: true } );
+
+			setPermissionScopeError( {
+				code: ERROR_CODE_MISSING_REQUIRED_SCOPE,
+				message: __(
+					'Additional permissions are required to enable enhanced measurement for the selected web data stream.',
+					'google-site-kit'
+				),
+				data: {
+					status: 403,
+					scopes,
+					skipModal: true,
+					redirectURL: global.location.href,
+				},
+			} );
+			return;
+		}
+
+		await commonSubmitChanges();
+	};
+
+	// If the user lands back on this component with autoSubmit and the edit scope,
+	// resubmit the form.
+	useEffect( () => {
+		async function handleAutoSubmit() {
+			// Auto-submit should only auto-invoke once.
+			setValues( FORM_SETUP, { autoSubmit: false } );
+
+			await commonSubmitChanges();
+		}
+
+		if ( autoSubmit && hasEditScope ) {
+			handleAutoSubmit();
+		}
+	}, [
+		hasEditScope,
+		setValues,
+		commonSubmitChanges,
+		onSubmitSuccess,
+		autoSubmit,
+	] );
+
+	const description = hasEditScope
+		? __(
+				'Enable enhanced measurement in Analytics to automatically track metrics like file downloads, video plays, form interactions, etc. No extra code required.',
+				'google-site-kit'
+		  )
+		: __(
+				'Enable enhanced measurement in Analytics to automatically track metrics like file downloads, video plays, form interactions, etc. No extra code required — you’ll be redirected to give permission for Site Kit to enable it on your behalf.',
+				'google-site-kit'
+		  );
+
+	return (
+		<BannerNotification
+			id="googlesitekit-enhanced-measurement-activation-banner"
+			className="googlesitekit-enhanced-measurement-setup-banner"
+			title={ __(
+				'Understand how visitors interact with your content',
+				'google-site-kit'
+			) }
+			description={ description }
+			learnMoreLabel={ __( 'Learn more', 'google-site-kit' ) }
+			learnMoreURL={ documentationURL }
+			ctaComponent={
+				<SpinnerButton
+					onClick={ handleSubmitChanges }
+					isSaving={ isSaving }
+				>
+					{ __( 'Enable now', 'google-site-kit' ) }
+				</SpinnerButton>
+			}
+			footer={
+				<p className="googlesitekit-enhanced-measurement-setup-banner__footer-notice">
+					{ __(
+						'You can always add/edit this in the Site Kit Settings',
```

Comment:
This additional footer text was also added as a result of the same conversation on [Slack](https://10up.slack.com/archives/CFFRMC5DE/p1696002454202519).

Reply:
Update: This copy is now confirmed :+1: